SimG4Core/Application/interface/TrackingAction.h

```diff
@@ -35,6 +41,8 @@ class TrackingAction : public G4UserTrackingAction {
   CMSSteppingVerbose* steppingVerbose_;
   const G4Track* g4Track_;
   bool checkTrack_;
+  bool doFineCalo_;
+  std::set<unsigned int> primaryIDs_;
```

Comment:
Does this collection need to be sorted, and how large is it typically? depending on the answers to those questions, `unordered_set` might perform better

Reply:
I think this whole set isn't needed anymore, I found a work-around. I deleted it.